### PR TITLE
delegation policy initial commit

### PR DESCRIPTION
**Pull request by nullmames**
Refactored version of the delegation policy collaboration by the delegation SubDAO.

### CODE REVIEW

File: policy/delegations.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+Any entity running a validator node on the Juno Network that satisfies the minimum requirements outlined in section 2.1
+
+## 1.5 How can a validator apply for a delegation?
+
+By submitting a delegation request [here](https://forms.gle/jkGKcxud2vswEA7Y8). The form will only be open for submissions during delegation application collection periods. These periods will be announced on the @JunoNetwork Twitter account.
+
+## 1.6 Delegation epoch
+
+Delegations will be made for a set time period. This period is known as an epoch. The epoch will be marked by a Juno Network block number which will be provided after applications are scored and delegations are made. The epoch length is determined by the SubDAO.
```
Comment (ShayneRarma) [inline policy/delegations.md:27]: Should we specify what an "epoch" is? I couldn't see a duration specified.

**Comment by nullmames** [comment]
I have not included the following two points:
- The Juno Delegation subDAO reserves the right to completely exclude validators from the program if valid evidence of harmful behaviour is presented. The final determination is at the absolute discretion of the Juno Delegations SubDAO.
- Activities that are already being paid for in some other way will not be considered within this JDP.

I would like to clarify before including.

File: policy/delegations.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
+# 1. Juno Delegation Program
+
+## 1.1 Juno Delegations SubDAO Inception
+
+The Juno Delegation Program (JDP) is part of the Juno Delegations SubDAO, which was established through Juno governance according to [proposal #50](https://www.mintscan.io/juno/proposals/50 "Juno Network Governance Proposal no. 50 on Mintscan").
+
+## 1.2 Why do we have a delegation program?
+
+The three core components of any distributed proof of stake blockchain are delegators, validators and the founding team. As the blockchain matures, additional roles are required to assist with community engagement and support to build on foundations set from genesis. Typical members/roles added include developers, marketing managers, and community managers. As Juno is a DAO-based blockchain, managing delegations to validators from the Juno Development Fund has been (primarily) moved from C-Root to the Juno Delegation SubDAO.
```
Comment (ShayneRarma) [inline policy/delegations.md:9]: "moved from C-Root" 

"moved from the founding team (aka "C-Root")

Just for future references incase there is a name change etc.

File: policy/delegations.md (hunk 1 above)
Comment (nullmames) [reply policy/delegations.md:27]: We could do this as a vote in the DAO.

IMO, we should be looking at minimum of 3 months initially, and then 6 months once we have more established data from the "previous epoch", at the moment we dont have any data for uptime, etc.

Comment (ShayneRarma) [reply policy/delegations.md:27]: Yep, happy to vote on the first duration. I think it should be somewhere in the doc (once we've voted) to inform the validators of the initial delegation period for the first round. At the moment, it's open-ended.

Comment (nullmames) [reply policy/delegations.md:27]: Agreed

Comment (nullmames) [reply policy/delegations.md:27]: https://daodao.zone/dao/juno1rw92sps9q4mm7ll3x9apnunlckchmn3v7cttchsf48dcdyajzj2sajfxcn/proposals/A3

File: policy/delegations.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+***2.1.2 (a)*** a minimum of one (1) form of contact information (e.g Twitter, Telegram, Email etc).
+
+***2.1.2 (b)*** a security contact. A verification email will be sent to the address provided for confirmation.
+
+***2.1.2 (c)*** a logo/avatar has been submitted to the Mintscan explorer GitHub repository and merged.
+
+***2.1.3 Maximum commission*** – The validator must have a commission rate lower than or equal to 10%.
+
+***2.1.4 Maximum rank*** – The validator must be ranked outside the top 10 validators, ranked by bonded voting power.
```
Comment (ShayneRarma) [inline policy/delegations.md:78]: ***2.1.4 Maximum rank*** – The validator must be ranked outside the top 10 validators, ranked by bonded voting power.

Should we add "at the time of application submission/review."

Primarily for the ones ranked 8-12 who are most likely to move in/out of the minimum criteria.

File: policy/delegations.md (hunk 1 above)
Comment (nullmames) [reply policy/delegations.md]: Epoch period has been updated.
https://github.com/CosmosContracts/delegations/pull/1/commits/a66ea4ebc133a41110469f9e140e5341e6b3cf80

File: policy/delegations.md (hunk 2 above)
Comment (nullmames) [reply policy/delegations.md]: Update has been made

**Pull request merged**